lib/flat_search/olx_scraper.ex

```diff
@@ -27,70 +27,75 @@ defmodule OlxScraper do
     res = Crawly.fetch(url)
     {:ok, document} = Floki.parse_document(res.body)
 
-    price =
-      document
-      |> Floki.find("h3, zł")
-      |> get_nested_element()
-      |> str_to_num()
+    flat_record = %{
+      price: get_price(document),
+      additional_price: get_additional_price(document),
+      title: get_title(document),
+      # description: get_description(document),
```

Comment:
should this still be commented out since you out commented the rest?

Reply:
Yeah, I don't wanna forget to add description later on, rest can be uncommented since description is not required and can be null in the db